src/Credentials/ServiceAccountCredentials.php

```diff
@@ -99,6 +99,11 @@ class ServiceAccountCredentials extends CredentialsLoader implements
      */
     private $jwtAccessCredentials;
 
+    /**
+     * @var string|null
```

Comment:
It seems this is needed to preserve backwards compatibility for the case of manually created CredentialsWrapper by users.